README.md

```diff
@@ -1,2 +1,18 @@
 # gargantua
 This repository hosts the core code for Gargantua, the megalith backend component of Hobby Farm. Currently, Hobby Farm is being built as a monolith, but will be transitioned into a proper microservices architecture one day™
+
+
+Gargantua components:
+Gargantua is based on K8S controllers, and provisions the following controllers to manage various aspects of HobbyFarm.
+- Terraform Provsioner Controller
+- VMClaim Controller
+- VMSet Controller
+- Environment Controller
+- ScenarioSession Controller
+
+Their relationship is better illustrated with the following ERD:
+
+![](./artifacts/diagrams/Workflow.png)
+
+**To do:**
+- helm chart
```

Comment:
There's a helm chart being developed here: https://github.com/boxboat/hobbyfarm

We'd like to get it moved to the hobbyfarm GitHub group at some point.

Reply:
Dropped the reference and I did see the helm chart.